script.js

```diff
@@ -1,46 +1,221 @@
+  ["Tab", "q", "w", "e", "r", "t", "y", "u", "i", "o", "p", "[", "]", "\\", "Del"],
+  ["CapsLock", "a", "s", "d", "f", "g", "h", "j", "k", "l", ";", "'", "Enter"],
+  ["Shift", "z", "x", "c", "v", "b", "n", "m", ",", ".", "/", "▲", "Shift"],
+  ["Ctrl", "Win", "Alt", " ", "Alt", "◄", "▼", "►", "Ctrl"],
 ];
-function addKeys(row,n){
-    for(let i = 0; i < allKeys[n].length; i++){
-        let keyInfo = allKeys[n][i];
 
-        let key = document.createElement('div');
-        
+function addKeys(row, n) {
+  for (let i = 0; i < keyCodes[n].length; i += 1) {
+    const key = document.createElement("div");
+    key.classList.add("key");
+    key.classList.add(keyCodes[n][i]);
+
+    const ru = document.createElement("div");
+    ru.classList.add("ru");
+
+    const ruKeyCaseUp = document.createElement("div");
+
+    ruKeyCaseUp.classList.add("caseUp");
+    ruKeyCaseUp.innerText = ruCaseUp[n][i];
+    ru.append(ruKeyCaseUp);
+
+    const ruKeyCaseDown = document.createElement("div");
+    ruKeyCaseDown.classList.add("caseDown");
+    ruKeyCaseDown.innerText = ruCaseDown[n][i];
+    ru.append(ruKeyCaseDown);
+
+
+    const en = document.createElement("div");
+    en.classList.add("en");
+
+    const enKeyCaseUp = document.createElement("div");
+    enKeyCaseUp.classList.add("caseUp");
+    enKeyCaseUp.innerText = enCaseUp[n][i];
+    en.append(enKeyCaseUp);
+
+    const enKeyCaseDown = document.createElement("div");
+    enKeyCaseDown.classList.add("caseDown");
+    enKeyCaseDown.innerText = enCaseDown[n][i];
+    en.append(enKeyCaseDown);
+
+    key.append(ru);
+    key.append(en);
+    row.append(key);
+  }
+}
+// add rows to keyboard
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
+for (let i = 0; i < 5; i += 1) {
+  const row = document.createElement("div");
+  row.className = "row";
+  addKeys(row, i);
+  keyboard.append(row);
+}
+// UpperCase and Language
+const enKeys = document.querySelectorAll(".en");
+const ruKeys = document.querySelectorAll(".ru");
+const upKeys = document.querySelectorAll(".caseUp");
+const downKeys = document.querySelectorAll(".caseDown");
+
+upKeys.forEach((el) => el.classList.toggle("hidden"));
+
+function changeLang() {
+  let lang = localStorage.getItem("lang");
+  if (!lang) lang = "ru";
+  if (lang === "ru") {
+    enKeys.forEach((el) => el.classList.remove("hidden"));
+    ruKeys.forEach((el) => el.classList.add("hidden"));
+    localStorage.setItem("lang", "en");
+  } else {
+    enKeys.forEach((el) => el.classList.add("hidden"));
+    ruKeys.forEach((el) => el.classList.remove("hidden"));
+    localStorage.setItem("lang", "ru");
+  }
+}
+function hideLangKeys() {
+  let lang = localStorage.getItem("lang");
+  if (!lang) lang = "ru";
+  if (lang === "ru") {
+    enKeys.forEach((el) => el.classList.add("hidden"));
+  } else {
+    ruKeys.forEach((el) => el.classList.add("hidden"));
+  }
+}
+hideLangKeys();
+function caps() {
```

Comment:
`changeCase` name is more appropriate here. Similar to `changeLang`.
Function name should start from a verb.